.ci/ci_job_flags.sh

```diff
@@ -93,13 +96,32 @@ case "${CI_JOB}" in
 	export KATA_HYPERVISOR="qemu"
 	export KUBERNETES="no"
 	;;
-"CRI_CONTAINERD"|"CRI_CONTAINERD_K8S")
+"CRI_CONTAINERD"|"CRI_CONTAINERD_K8S"|"CC_CRI_CONTAINERD")
 	# This job only tests containerd + k8s
 	init_ci_flags
 	export CRI_CONTAINERD="yes"
 	export CRI_RUNTIME="containerd"
 	export KATA_HYPERVISOR="qemu"
-	[ "${CI_JOB}" == "CRI_CONTAINERD_K8S" ] && export KUBERNETES="yes"
+	case "${CI_JOB}" in
+		"CRI_CONTAINERD_K8S")
+			export KUBERNETES="yes"
+			;;
+		"CC_CRI_CONTAINERD")
+			# Export any CC specific environment variables
+			export KATA_BUILD_CC="yes"
+			export SECCOMP=yes
```

Comment:
Is sccomp really CC specific? Why no quotes around `yes` here?

Reply:
good question @fitzthum . I blindly copied that from `ccv0.sh` script. @stevenhorsman is seccomp required for CC?

Reply:
So from `rootfs.sh`
https://github.com/kata-containers/kata-containers/blob/dbb0c675236c520d65a16acbf44978fec9efabda/tools/osbuilder/rootfs-builder/rootfs.sh#L25-L27
it looks like the agent is build with `SECCOMP="yes"` by default, so we probably don't need to set it explicitly for CC targets.

Reply:
cool. @stevenhorsman @fitzthum so I opened a PR to change that on CCv0 branch (pass through CI to ensure it doesn't break): https://github.com/kata-containers/tests/pull/4708

Once it is merged then I will update this PR.